Formula/feroxbuster.rb

```diff
@@ -0,0 +1,19 @@
+# typed: false
+# frozen_string_literal: true
+
+# :-)
+class Feroxbuster < Formula
+  desc "Fast, simple, recursive content discovery tool written in Rust. 🦀"
+  homepage "https://github.com/epi052/feroxbuster"
+  url "https://github.com/epi052/feroxbuster/releases/download/v1.1.0/x86_64-macos-feroxbuster.tar.gz"
+  sha256 "cebccff81d0c949918e112032a5daeafaec09b79f855e1cd3df71069359c782d"
+  license "MIT"
+
+  def install
+    bin.install "feroxbuster"
+  end
+
+  test do
+    assert true
```

Comment:
This isn't a test.

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.